content/tutorial.md

```diff
@@ -50,7 +50,7 @@ Follow the links below to install Quire:
 
 ## 3. Create a New Project
 
-To start a new Quire project, open your command-line shell and type `quire new my-project`. Quire will download a new starter project into a folder named “my-project” in your home directory. If you are using the Beta version of Quire, you may need to enter your GitHub username and password twice during the download process: once for the starter kit and again for the starter theme.
+To start a new Quire project, open your command-line shell and type `quire new my-project`. Quire will download a new starter project into a folder named “my-project” in your home directory.
```

Comment:
Is it worth explaining what we mean by "starter project" (i.e. sample publication including content, images, and relevant metadata) or do you think it's self-explanatory?

Reply:
good idea. I'll add this